server/sonar-server/src/main/java/org/sonar/server/computation/component/DepthTraversalTypeAwareVisitor.java

```diff
@@ -43,17 +43,18 @@ public void visit(Component component) {
     }
 
     if (order == Order.PRE_ORDER) {
-      visitAny(component);
+      visitNode(component);
```

Comment:
`visitNode` is private, `visitAny` is public and intended to be override
when a subclass would override `visitNode`, it would break support for calling all the public `visit*` method unless it called `super.visitNode`